src/mixins/filing-template-mixin.ts

```diff
@@ -127,10 +127,16 @@ export default class FilingTemplateMixin extends DateMixin {
         contactPoint: this.getContactPoint,
         nameRequest: this.getNameRequest,
         offices: this.getOfficeAddresses,
+        parties: undefined, // parties are added below
         type: this.getCorrectionErrorType
       }
     }
 
+    // delete invalid NAICS properties
+    delete filing.business.naicsCode
+    delete filing.business.naicsDescription
+    delete filing.business.naicsKey
```

Comment:
These properties are in the business object but not used for Incorporation Application filings.